packages/core/src/typings/application-command.ts

```diff
@@ -44,8 +51,11 @@ export declare namespace ApplicationCommand {
         description: string;
         options?: Array<Option.Structure>;
         default_member_permissions: string | null;
+        /** @deprecated Use {@link GlobalStructure.contexts} instead */
         dm_permission: boolean;
```

Comment:
Since its deprecated lets just remove it, since lilybird is still in development and doesnt have a 1.0 release yet we should just ignore all the deprecated stuff